### ISSUE TITLE
Question about the Dataset

### ISSUE THREAD

**Issue by dbjhbyun** [kaixin96/PANet#26]
Hi,

I was looking through the code but couldn't figure out what "scribble" indicates.
From dataloaders/customized.py
Line 103
support_scribbles = [[paired_sample[cumsum_idx[i] + j]['scribble'] for j in range(n_shots)]
                         for i in range(n_ways)]

Could you clarify what the support_scribbles mean? Also, what is "image_t"
From dataloaders/customized.py
Line 93
support_images_t = [[paired_sample[cumsum_idx[i] + j]['image_t'] for j in range(n_shots)]
                        for i in range(n_ways)]

Is it the segmentation target mask of the given image? While, label indicates the class_index of the mask?

Thanks in advance,

**Comment by kaixin96** [comment]
Hi @dbjhbyun, `scribble` refers to the scribble annotations mentioned in Section 4.4 of our paper.

I must apologize for the confusing variable name `image_t`. I use it to denote the images after transformation (flip or resize) but before Pytorch's `toTensor` (which scales and normalizes the input with ImageNet mean and std). They are useful when you want to manually check the images or do some visualization.

Thank you.
